### ISSUE TITLE
Incorrect warning about MySQL not being widely tested while actually using SQLite

### ISSUE THREAD

**Issue by kristapsk** [btcpayserver/btcpayserver#1695]
Not a big issue, but when starting current master with default settings (`./start.sh` without any parameters), got incorrect warning at the startup:

```
info: Configuration:  SQLite DB used (Data Source=/home/user/.btcpayserver/Main/sqllite.db)
warn: Configuration:  MySQL is not widely tested and should be considered experimental, we advise you to use postgres instead.
```

Guess it should be changed to "whatevernonpostgresdb is not widely tested...".

**Comment by NicolasDorier** [comment]
SQLite is every worsely tested than MySql. I need to fix the message.

**Comment by NicolasDorier** [comment]
fixed in 1.0.5.2
